inscriptions/ISic000178.xml

```diff
@@ -189,21 +198,20 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#JP">
                 <ab>
-                    <lb n="1"/><expan><abbr>M</abbr><ex>arco</ex></expan> Granio <expan><abbr>M</abbr><ex>arci</ex></expan> <expan><abbr>f</abbr><ex>ilio</ex></expan>
-                    <lb n="2"/><expan><abbr>Quir</abbr><ex>ina tribu</ex></expan> Dextro <expan><abbr><hi rend="small">v</hi></abbr><ex>ixit</ex></expan> <expan><abbr><hi rend="small">a</hi></abbr><ex>nnis</ex></expan> <hi rend="small"><num value="50">L</num></hi>
-                    <lb n="3"/>Cestia Pulchella
+                    <lb n="1"/><persName type="attested"><expan><abbr>M</abbr><ex>arco</ex></expan></persName> <persName type="attested">Granio</persName> <persName type="attested"><expan><abbr>M</abbr><ex>arci</ex></expan></persName> <expan><abbr>f</abbr><ex>ilio</ex></expan>
+                    <lb n="2"/><expan><abbr>Quir</abbr><ex>ina tribu</ex></expan> <persName type="attested">Dextro</persName> <expan><abbr><hi rend="small">v</hi></abbr><ex>ixit</ex></expan> <expan><abbr><hi rend="small">a</hi></abbr><ex>nnis</ex></expan> <hi rend="small"><num value="50">L</num></hi>
+                    <lb n="3"/><persName type="attested">Cestia</persName> <persName type="attested">Pulchella</persName>
                     <lb n="4"/>fecit sibi et viro suo
                 </ab>
             </div>
             <div type="apparatus">
-                <listApp><app><note>Text after ILTermini</note></app></listApp>
+                <listApp><app><note>Text from autopsy</note></app></listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#AA">
+                <p>To Marcus Granius Dexter, of tribe Quirina, son of Marcus, who lived for 50 years. Cestia Pulchella built (this) for herself and her husband.</p>
             </div>
-            <div type="commentary">
-                <p><!--commented out pending revision-->
-                    <!--set up by Cestia Pulchella for herself and her husband M.Granius M.f.Quir(ina tribu) Dexter - a latin cogn.; B wonders whether he's African.--></p>
+            <div type="commentary"> 
+                <p>Cestiae are relatively common in Thermae Himeraeae. Roman citizens of tribe Quirina are attested elsewhere in Thermae Himeraeae, see <ref target="http://sicily.classics.ox.ac.uk/inscription/ISic000169">ISic000169</ref> and <ref target="http://sicily.classics.ox.ac.uk/inscription/ISic000187">ISic000187</ref>. Bivona (1994) hypothesizes an African origin of M. Granius Dexter, and posits that the small letters squeezed into the end of line 2 are a later addition, perhaps added after Dexter's death. Bivona also suggests on palaeographical grounds that lines 3-4 are different in character, and are less rigid, but this is not apparent after autopsy. The tribue Quirinia</p>
```

Comment:
something funny in the final sentence? On tribals, for what it's worth, Prag, J. R. W. (2010). Sicilia Romana tributim discripta. In M. Silvestrini (Ed.), Le tribù romane. Atti della XVIe Rencontre sur l’épigraphie (Bari 8-10 ottobre 2009) (pp. 305–311). Edipuglia.